src/main/resources/assets/ingameinfo/lang/de_DE.lang

```diff
@@ -1,45 +1,284 @@
+# https://github.com/ReneMuetti/InGame-Info-XML
```

Comment:
My sanitation script will wipe this the next time I run it, so it will only add noise. And git already keeps a history of commits including the authors.